lms/djangoapps/mailing/management/commands/mailchimp_id.py

```diff
@@ -14,30 +15,24 @@ class Command(BaseCommand):
     Given a mailchimp key, validates that a list with that key
     exists in mailchimp.
     """
-    args = '<mailchimp_key web_id>'
     help = 'Get the list id from a web_id'
 
-    option_list = BaseCommand.option_list + (
-        make_option('--key', action='store', help='mailchimp api key'),
-        make_option('--webid', action='store', dest='web_id', type=int,
-                    help='mailchimp list web id'),
-    )
-
-    def parse_options(self, options):
-        """Parses `options` of the command."""
-        if not options['key']:
-            raise CommandError('missing key')
-
-        if not options['web_id']:
-            raise CommandError('missing list web id')
-
-        return options['key'], options['web_id']
+    def add_arguments(self, parser):
+        parser.add_argument('--key',
+                            required=True,
```

Comment:
I recall @bmedx commenting that `call_command` didn't play nicely with "required options"; do we not have tests for these which use that?

Reply:
In the student management command PR (https://github.com/edx/edx-platform/pull/16384), the solution was to ultimately use required options, but hack around the call_command limitation (bug?) by passing a list of shell command tokens rather than standard keyword args.  At least, that was my interpretation.

Reply:
And in this PR the commands I'm touching don't have tests so there's no call_command hacking to do.